tests/integration/targets/utils_keep_keys/vars/main.yaml

```diff
@@ -22,3 +22,22 @@ keep_default:
           is_enabled: true
     - interface_name: eth2
       is_enabled: false
+
+l1:
```

Comment:
@roverflow can you please create a new file called complex.yaml which has the output of all the complex integration tests, as the format of our and these tests are different its better if we keep them in a different file, you can refer this [PR](https://github.com/ansible-collections/ansible.utils/pull/200) for that.
CC: @NilashishC

Reply:
+1 to that.